cou/apps/base.py

```diff
@@ -363,7 +369,8 @@ async def _verify_workload_upgrade(self, target: OpenStackRelease, units: list[U
 
         if units_not_upgraded:
             raise ApplicationError(
-                f"Cannot upgrade units '{', '.join(units_not_upgraded)}' to {target}."
+                f"Cannot upgrade unit(s) '{', '.join(units_not_upgraded)}' to {target}. "
+                "Try again in some minutes to see if the unit(s) have been upgraded."
```

Comment:
nit: technically if we have reached this line the upgrade process itself did not fail, so it's not us who were unable to upgrade the units.
```suggestion
                f"Unit(s) '{', '.join(units_not_upgraded)}' did not complete the upgrade to {target}. "
                "Some local processes may still be executing; you may try re-running COU in a few minutes."
```